provider/resources.go

```diff
@@ -345,8 +345,12 @@ func preConfigureCallback(vars resource.PropertyMap, c shim.ResourceConfig) erro
 	config.SharedConfigFiles = []string{configPath}
 
 	if _, err := awsbase.GetAwsConfig(context.Background(), config); err != nil {
-		return fmt.Errorf("unable to validate AWS credentials " +
-			"- see https://pulumi.io/install/aws.html for details on configuration")
+
+		return fmt.Errorf("unable to validate AWS credentials. Make sure you have: \n\n" +
+			" \t • Set the AWS_SECRET_ACCESS_KEY \n" +
+			" \t • Set the AWS_ACCESS_KEY_ID \n" +
+			" \t • Set your AWS region using `pulumi config set aws:region <region>`, e.g `us-west-2` \n\n" +
```

Comment:
@guineveresaenger I'd like to make sure to convey that we integrate with the configuration that a user might already have in their .aws directory. I think suggesting `aws configure` would help imply that we support `aws sso login`

Reply:
Fair point - I made the checklist more generic and suggested `aws configure`. WDYT?

Reply:
Folks, the user may have already used `aws configure` and are unsure of what we expect of the user. I don't think `aws configure` is correct herre